src/main/java/duke/task/TaskList.java

```diff
@@ -0,0 +1,95 @@
+package duke.task; 
+
+import java.util.List;
+import duke.DukeException;
+import duke.util.Parser;
+
+public class TaskList {
+    private List<Task> tasks;
+
+    public TaskList(List<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    public void addTask(String input) throws DukeException {
+        String[] split = input.split(" ", 2);
+
+        if (split.length < 2) {
+            throw new DukeException("\tSorry! The description of a todo cannot be empty :')");
+        } else {
+            String keyword = split[0];
+            String description = split[1];
+
+            switch (keyword) {
+            case "todo":
+                Task task = new ToDo(description, false);
+                this.tasks.add(task);
+                printAddedConfirmation(task);
+                break;
+            case "deadline":
+                String[] splitSlash = description.split(" /by ");
+                if (splitSlash.length != 2) {
+                    throw new DukeException("\tPaise! :') Please use the format: deadline <task> /by <time>" +
+                            "\n\t\t*time format: <yyyy-mm-dd> or  <yyyy-mm-dd HH:mm>");
+                }
+                task = new Deadline(splitSlash[0], false, Parser.parseDate(splitSlash[1]));
+                this.tasks.add(task);
+                printAddedConfirmation(task);
+                break;
+            case "event":
+                splitSlash = description.split(" /at ");
+                if (splitSlash.length != 2) {
+                    throw new DukeException("\tPaise! :') Please use the correct format: event <task> /at <time> +" +
+                            "\n\t\t*time format: <yyyy-mm-dd> or  <yyyy-mm-dd HH:mm>");
```

Comment:
Consider breaking before the `+` operator instead as mentioned in the style guide